src/main/java/io/github/thebusybiscuit/cscorelib2/blocks/ChunkPosition.java

```diff
@@ -134,6 +135,19 @@ public static long getAsLong(int x, int z) {
         return (((long) x) << 32) | (z & 0xFFFFFFFFL);
     }
 
+    /**
+     * Decompress a ChunkPosition into an int[].
+     * @param position The compressed ChunkPosition
+     * @return Returns a 2 length int[] where the x coordinate is at index 0 and z at index 1.
+     *
+     */
+    public static int[] decompress(long position) {
+        int x = (int) (position >> 38);
+        int y = (int) (position & 0XFFF);
+        int z = (int) (position << 26 >> 38);
+        return new int[]{x, y, z};
```

Comment:
"2 length int[]", int[] has a length of 3.

:eyes:

Reply:
Sssshh no one saw the copypasta 👀